src/config.ts

```diff
@@ -1,11 +1,20 @@
 import { CheckpointConfig } from '@snapshot-labs/checkpoint';
 import Schnaps from './abis/Schnaps.json';
 
+/** Infura API key used by default for network nodes. */
+export const DEFAULT_INFURA_API_KEY =
+  process.env.INFURA_API_KEY || '46a5dd9727bf48d4a132672d3f376146';
+
 const CONFIG = {
+  sepolia: {
+    networkNodeUrl: `https://sepolia.infura.io/v3/${DEFAULT_INFURA_API_KEY}`,
```

Comment:
We should use brovider instead like
```suggestion
    networkNodeUrl: 'https://rpc.brovider.xyz/11155111',
```

Reply:
ah, used the config like we do on sx-api : https://github.com/snapshot-labs/sx-monorepo/blob/4fe1f1cce8572ceb303d17a21ea02c478b193d16/apps/api/src/starknet/config.ts#L8-L10